lib/productions/type.js

```diff
@@ -89,8 +89,9 @@ function union_type(tokeniser, type) {
   const ret = new Type({ source: tokeniser.source, tokens });
   ret.type = type || null;
   while (true) {
+    if(tokeniser.probe("any")) tokeniser.error("Type `any` cannot be included in a union type");
```

Comment:
I'm sorry to say this, but I found a bug which blocks this change (#436), which is why `union-any.txt` is now gone empty. Could you revert this change to match what you did in #431? This includes the Promise line below, because this way we can't prevent `[ExtAttr] Promise` being included in a union. Sorry to confuse you 😢